extensions/functions_arithmetic.yaml

```diff
@@ -1369,6 +1369,109 @@ aggregate_functions:
         ordered: true
         return: LIST?<any>
 
+  - name: "bit_or"
+    description: >
+      Calculates the bitwise OR of all values in set of values.
+    impls:
+      - args:
+          - name: x
+            value: i8
+        nullability: DECLARED_OUTPUT
+        return: i8
```

Comment:
The output is non-nullable, so what do these aggregates do when there is no input?